scripts/get-latest-rhocp-repo.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Following script queries RHOCP repositories to get latest available
+# for version of MicroShift from current branch.
+# It expect system to be registered (i.e. entitlement exists).
+#
+# We cannot use branch version (or sometimes even previous minor one)
+# because repositories are only usable after the release.
+# Accessing them before the release results in 403 error.
+#
+# Output is just the minor (Y) version number.
+
+set -o pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+REPOROOT="$(cd "${SCRIPTDIR}/.." && pwd)"
+
+if ! sudo subscription-manager status >&/dev/null; then
+    >&2 echo "System must be subscribed"
+    exit 1
+fi
+
+cacert="/etc/rhsm/ca/redhat-uep.pem"
+cert=$(find /etc/pki/entitlement -iname '*.pem' -not -iname '*-key.pem')
+key=$(find /etc/pki/entitlement -iname '*-key.pem')
+
+current_minor=$(make -C "${REPOROOT}" debug | grep "MINOR:" | cut -d':' -f2)
+stop=$(( current_minor - 3 ))
+
+for ver in $(seq "${current_minor}" -1 "${stop}"); do
+    repository="https://cdn.redhat.com/content/dist/layered/rhel9/$(uname -m)/rhocp/4.${ver}"
+    exit_code=$(curl \
+        --silent \
+        --location \
+        --output /dev/null \
+        --write-out "%{http_code}" \
+        --cacert "${cacert}" \
+        --cert "${cert}" \
+        --key "${key}" \
+        "${repository}/os/repodata/repomd.xml")
+
+    if [[ "${exit_code}" == "200" ]]; then 
+        echo "${ver}"
```

Comment:
I wonder if we need to just output the whole version, i.e. `4.14`? 
It looks like we're appending `4.` everywhere it's used.

Reply:
By printing only the minor, we can easily manipulate it further. For example, on `release-4.14` branch we're enabling RHOCP 4.13 and 4.14. So if we have similar situation in future, we can easily get minor-1 without too much problem.
On `r-4.15` it will happen when `rhocp-4.15` will be published and we'll switch to it to test upgrade from released RPMs, but without `rhocp-4.14` will only use RCs and they will be long outdated by then.

Reply:
This happens because https://github.com/openshift/microshift/blob/main/test/bin/get_rel_version_repo.sh, for released RPMs, only checks system repositories - if rhocp-4.14 is not enabled, it will take version from openshift mirror